packages/common/src/api/tan-query/comments.ts

```diff
@@ -513,6 +512,8 @@ export const usePostComment = () => {
       args.newId = newId
       const newComment: Comment = {
         id: newId,
+        entityId: encodeHashId(trackId)!,
```

Comment:
I'm surprise `encodeHashId` still exists. Can you use the `Id.parse()` function from SDK instead?